CefSharp/IBrowserHost.cs

```diff
@@ -11,6 +11,14 @@ public interface IBrowserHost : IDisposable
     {
         void StartDownload(string url);
         void Print();
+        /// <summary>
+        /// Prints the current browser contents to the PDF file and executes a callback
+        /// on completion. The caller is responsible for deleting the file when done.
+        /// </summary>
+        /// <param name="path">PDF file location.</param>
+        /// <param name="settings">Settings.</param>
+        /// <param name="callback">The callback executed on completion.</param>
+        void PrintToPDF(string path, CefSharpPdfPrintSettings settings, IPrintToPdfCallback callback);
```

Comment:
To me it seems like that path param will be exactly whatever you specify in PrintToPDF so I don't think we need that at all. How about returning a simple Task<bool>, or better just a Task and signal an exception on failure?